### DISCUSSION TITLE
Dedicated User

### DISCUSSION THREAD

**Discussion by 2q2code** [creativeprojects/resticprofile#437] [Q&A] [answered]
I just want to double check this. I have a dedicated `restic` user that I use to run both `restic` and `resticprofile`. The user is the only one that has permission to use `restic`, as I apply CAP_DAC_SEARCH_READ to it to allow it to back up all files regardless of permissions. It also has permission to perform dumps of all databases. When scheduling, I have added a drop-in for systemd to set the user and group directly, amongst other things. How should I set `permissions` in the profile?

**Comment by creativeprojects** [comment, marked as answer]
Sorry for the late reply...

The systemd service will need to start as `root` to be able to change the user and group, so the permission should be set to `system`

**Reply by 2q2code** [reply]
Brilliant. Thanks Fred.
